sdk/ledger-service/http-json/src/it/scala/http/HttpServiceIntegrationTest.scala

```diff
@@ -227,6 +227,50 @@ abstract class HttpServiceIntegrationTest
       } yield result should ===("Bob invoked IIou.Transfer")
     }
 
+    // This is a hacky way to determine the package id of a dar,
+    // but might suffice for tests for now.
+    def packageIdOfDar(dar: File): Future[String] = Future {
```

Comment:
I don't think you need to go so far with getting a package ID, it would be possible to get it from the result from a postCreateCommand, no? (like just creating a contract without a package id, from result getting a package id, then using that)

Reply:
Yes, that would probably work in this case. The bigger context is I'm planning to re-use this for a follow-on PR where I'll test upgrade scenarios where you'll have more than one package id per (module, entity) pair, so omitting the package id/ref wouldn't work.